cookietemple/create/templates/available_templates.yml

```diff
@@ -11,7 +11,7 @@ cli:
     python:
         name: Python Commandline Package
         handle: cli-python
-        version: 1.0.0
+        version: 2.0.0
```

Comment:
u sure

Reply:
WIP, will change this, right now I'm working on this.